plugin/src/main/java/org/openrewrite/gradle/AbstractRewriteTask.java

```diff
@@ -176,17 +178,30 @@ protected ResultsContainer listResults() {
                     .map(AbstractRewriteTask::toRealPath)
                     .collect(toList());
 
+            String javaRuntimeVersion = System.getProperty("java.runtime.version");
+            RewriteJavaMetadata javaMetadata =  sourceSets.get(sourceSet);
+            JavaProvenance javaProvenance = rewrite.javaProvenanceBuilder()
+                    .projectName(getProject().getName())
+                    .buildToolVersion(GradleVersion.current().getVersion())
+                    .sourceSetName(sourceSet.getName())
+                    .vmRuntimeVersion(javaRuntimeVersion)
+                    .vmRuntimeVersion(System.getProperty("java.vm.vendor"))
```

Comment:
This looks accidental. You probably didn't mean to set the same property twice with different values

Reply:
yep, should be vmVendor, good catch